app/services/finance/ecf/statement_calculator.rb

```diff
@@ -132,7 +132,7 @@ def total_for_uplift
       end
 
       def adjustments_total
-        total_for_uplift - clawback_deductions
+        -clawback_deductions
```

Comment:
Adjustments will now appear as negative in the summary table, do we want that value to change to that?

Reply:
if its negative zero this now shows it as unsigned zero with the new commit on the end